doc/level1.rst

```diff
@@ -6,22 +6,51 @@
 Autocrypt Level 1: Enabling encryption, avoiding annoyances
 ===========================================================
 
-Autocrypt makes it easy for people to encrypt email.  This document
-describes the basic capabilities required for a mail app to be
-Autocrypt-capable at Level 1.
-
-The design of Level 1 is driven by usability concerns and by the
-realities of incremental deployment. A user may mix both
-Autocrypt-enabled MUAs and traditional MUAs, and we'd
-like to avoid annoyances like unexpectedly unreadable mails while also
-supporting users who want to explicitly turn on encryption.
-
-For ease of implementation and deployment, Level 1 focuses on the use
-of Autocrypt on a single device.  We intend to :doc:`support
-multi-device synchronization (and other features) as part of Level
-2<next-steps>`.  We want to keep Level 1 simple enough that it's easy
-for developers to adopt it so we can drive efforts from real-life
-experiences as soon as possible.
+This specification is the result of maybe 2000 hours of talking, testing and
+coding iterations during 2017. Around 20 developers and many more users and
+trainers participated overall in Autocrypt sessions. The majority of these
+participants have engaged with the e-mail encryption space for a
+long time. There was overwhelming agreement that fresh approaches are needed
+and particularly ones that address usability concerns.
+
+Autocrypt aims to incrementally replace cleartext e-mail with end-to-end
+encrypted e-mail. It aims to sustainably grow the overall percentage of
+end-to-end encrypted messages relayed via the e-mail provider network.
+This differs from the traditional focus on maximizing the security of an
+individual mail communication. **Sometimes Autocrypt recommends to
+send cleartext mail even though encryption would be technically
+possible.** This is because of well-known annoyances around "i can't
+read your encrypted mail" situations. Autocrypt operates opportunistically
+and tries to stay out of the way of users who want to, first of all,
+communicate.
+
+Another difference to traditional approaches is that Autocrypt Level 1
+only defends against passive data collection. Protection against active
+adversaries (those which modify messages in transit) is the aim of
+future specifications. We share and support :rfc:`the new perspective
+stated in RFC7435 ("Opportunistic Security: Some Protection Most of the
+Time") <7435#section-1.2>`.
+
+The Level 1 specification makes it easy for users to encrypt. There are
+no changes required from e-mail providers and there are no dependencies
+on key servers. The spec describes the basic capabilities required for a
+mail app to be Autocrypt-capable at Level 1, allowing it to exchange
+end-to-end encrypted e-mails with other Autocrypt-capable mail apps. For
+those familiar with existing e-mail specs, we use the term "mail app"
+and "MUA" interchangeably throughout the doc.
```

Comment:
Is this necessary? We never use mail app anywhere else in the spec so far. These terms are also from the opposite spectrum of "specialized tech term" to "new and popular term in informal language", which can lead to unnecessary confusion. I'd prefer if we stuck to one term consistently.

Reply:
i was previsouly under the impression that we used mail app throughout the spec.  That isn't the case (anymore?).  I am fine with using MUA in the technical sections but i'd like to use a language in the very intro that does not alienate all the young folks who have never heart of MUAs. And i don't want to use the first paragraph to introduce/explain MUA.